### PR TITLE
Extra checks when configuring Avahi

### PR DESCRIPTION
**Pull request by srbarrios**
## What does this PR change?

Some tips when troubleshooting Avahi from my local to a deployed VM.

### CODE REVIEW

File: TROUBLESHOOTING.md [hunk 1]
```diff
@@ -30,6 +30,8 @@ Check that:
   - on SUSE systems check YaST -> Security and Users -> Firewall -> Zones -> public, "mdns" should appear on the list on the right
  - Avahi is installed and running
    - typically you can check this via systemd: `systemctl status avahi-daemon`
+   - be sure you don't have mixed versions: `rpm -qa | grep avahi`
+ - Your firewall allows `mdns` on libvirt zone
```
Comment (Bischoff) [inline TROUBLESHOOTING.md:34]: firewall is already mentioned above (but specifies "public" instead of "libvirt"

File: TROUBLESHOOTING.md [hunk 2]
```diff
@@ -30,6 +30,8 @@ Check that:
...
  - Avahi is capable of resolving the host you are trying to reach: `avahi-resolve -n uyuni.tf.local`
```
Comment (Bischoff) [inline TROUBLESHOOTING.md:35]: maybe `<host>` instead of `uyuni`

File: TROUBLESHOOTING.md [hunk 3]
```diff
@@ -30,6 +30,8 @@ Check that:
   - on SUSE systems check YaST -> Security and Users -> Firewall -> Zones -> public, "mdns" should appear on the list on the right
```
Comment (Bischoff) [inline TROUBLESHOOTING.md:30]: "libvirt" instead of "public" (or specify both with "or")

**Review by Bischoff — CHANGES_REQUESTED** [review]
thanks; some changes

**Review by Bischoff — APPROVED** [review]
perfect now, thanks

**Comment by moio** [comment]
Thanks @srbarrios !!!

**Pull request merged**